web-registry/src/pages/Projects/hooks/useProjectsSellOrders.utils.tsx

```diff
@@ -39,18 +38,3 @@ export const getPurchaseInfo = (
     },
   };
 };
-
-export const sortProjectsBySellOrdersAvailability =
```

Comment:
Removing this one because we have a sort file with a similar function. All sorts show available credits first now.